typescript/cli/src/commands/validator.ts

```diff
@@ -49,3 +64,58 @@ const addressCommand: CommandModuleWithContext<{
     process.exit(0);
   },
 };
+
+const preFlightCheckCommand: CommandModuleWithContext<{
+  chain: string;
+  validators: string;
+}> = {
+  command: 'check',
```

Comment:
maybe worth renaming the file from preFlightCheck.ts to check.ts too for consistency